### PR TITLE
Tiny UI changes

### PR DESCRIPTION
**Pull request by ghost**
Hey, I finally got to work on it. Just aligned some elements on mobile and desktop, like profile images in header and footer, tags section on mobile and desktop (there were big gaps between tags and article cards before)

### CODE REVIEW

File: src/components/navigation/index.less [hunk 1]
```diff
@@ -173,6 +173,11 @@
 
 .nav__social-icons-item-link {
   color: @clr-text;
+  display: flex;
+
+  svg {
```
Comment (vstanyshevskyy) [inline src/components/navigation/index.less]: Please follow BEM notation:
```
.nav__social-icons-item-link {
  color: @clr-text;
  display: flex;
  .social-icon {
    margin: auto;
  }
}
```
However, it's not good BEM anyway - we would need to pass the container down to SocialIcons component to make it right

File: src/components/tags/index.less [hunk 2]
```diff
@@ -1,5 +1,14 @@
 @import '../styleguide/variables.less';
```
Comment (vstanyshevskyy) [inline src/components/tags/index.less]: It's a cool idea you have to show tags list full-width for mobile devices 💪.
And BTW, there is a way to simplify the implementation:
1. Add this to `src/components/article-card/index.less`
```
.article-card--tags {
  margin-left: 0;
  width: ~"calc(50% - 15px)";
  @media screen and (max-width: @screen-sm-max) { 
    width: 100%;
  }
}
```

2. Revert the changes you've made to `.tags` in `src/components/tags/index.less`
3. Add `article-card article-card--tags` classnames back to `src/templates/listPage.js `

File: src/templates/index.less [hunk 3]
```diff
@@ -28,14 +28,15 @@
 }
 
 .tags--advice {
-  width: ~"calc(50% - 15px)";
+  // width: ~"calc(50% - 15px)";
```
Comment (vstanyshevskyy) [inline src/templates/index.less]: Feel free to delete any redundant code :)

**Review by vstanyshevskyy — CHANGES_REQUESTED** [review]
Well done @heyiamvika! Thanks for your attention to the details and creativity - I really like the idea of showing the tags full-width for mobile screens.

I left a few comments - those are minor things.

File: src/components/navigation/index.less (hunk 1 above)
Comment (heyiamvika) [reply src/components/navigation/index.less]: yep, you're right, I'm not very familiar with BEM yet, but I'll have a look. Thanks!
